tests/test_qvm.py

```diff
@@ -33,7 +33,6 @@
     "CZ 1 0\n"
     "MEASURE 1 ro[0]\n"
     "MEASURE 0 ro[1]\n"
-    "HALT\n"
```

Comment:
What was this for?

Reply:
There's a test in `test_qvm.py` where the program is compared for equality. Running locally with `quilc -P -S` and `qvm -S`, there wasn't a `HALT` instruction and this test failed. If `quilc` is run without the protoquil `-P` flag, it's possible that this `HALT` instruction would be included; I'll put this back the way it was.